src/main/java/org/broadinstitute/consent/http/util/gson/InstantTypeAdapter.java

```diff
@@ -16,14 +16,14 @@
         implements JsonSerializer<Instant>, JsonDeserializer<Instant> {
     @Override
     public JsonElement serialize(Instant src, Type srcType, JsonSerializationContext context) {
-        return new JsonPrimitive(src.toString());
+        return new JsonPrimitive(src.toEpochMilli());
```

Comment:
I changed this to make instants serialize dates as milliseconds, just like our current dates do. Is there any preference to using the more accurate ISO string format that the Instant's `toString()` has or the epoch milliseconds which mirrors our current implementation?

Reply:
I think this is the better choice right now. The complication would be in the UI if there are any calls that are expecting string values and I think we need to look at those instances individually and gauge the work required for updates.